### PR TITLE
[39] Display an error message if bundler failed to run

### LINKED ISSUE #39: Return an error if bundler is not present in the system
## Why
We want to let the user know that bundler was not found in the system when trying to create the gem.

## What
Add the `exception: true` option to the `Kernel#system` call and rescue the error to display a message to the end user.

### PR DESCRIPTION
**Pull request by juanmanuelramallo**
Resolves #39

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # master
 
+* Display an error message if bundler failed to run or is not present in the system
```
Comment (santib) [inline CHANGELOG.md:3]: 😱

File: lib/rsgem/tasks/create_gem.rb [hunk 2]
```diff
@@ -10,8 +10,13 @@ def initialize(context:)
       end
 
       def create
-        system shell_command
-        puts message
+        if system(shell_command)
+          puts message
+        else
+          puts "Failed to run `bundle gem'. "\
+               "Check bundler is installed in your system, or install it with `gem instal bundler'"
```
Comment (santib) [inline lib/rsgem/tasks/create_gem.rb]:
```suggestion
               "Check bundler is installed in your system, or install it with `gem install bundler`"
```

**Review by santib — APPROVED** [review]

**Review by juan-apa — APPROVED** [review]
Nice addition. However, in the documentation it says that the system() function returns `nil` if the command doesn't exist, and false if it has a non zero exit code. WDYT about making a distinction between those 2 possible error outcomes so that the user doesn't have to check if bundler is installed?

**Comment by juanmanuelramallo** [comment]
I guess there's a really small amount of chance that a user won't have bundler installed in their system. So this is actually more of a safeguard for rsgem not to run anything if bundler failed. 

I'd rather keep it simple by just stating that bundler failed or it's not installed, what do you think of that @juan-apa ?

**Comment by juan-apa** [comment]
> I guess there's a really small amount of chance that a user won't have bundler installed in their system. So this is actually more of a safeguard for rsgem not to run anything if bundler failed.
> 
> I'd rather keep it simple by just stating that bundler failed or it's not installed, what do you think of that @juan-apa ?

I see. nice 😄

**Review by jpascual1994 — APPROVED** [review]

**Pull request merged**